doc/CMakeLists.txt

```diff
@@ -19,20 +19,20 @@ find_program(SPHINX_EXECUTABLE
         DOC "Sphinx documentation generation tool (http://www.sphinx-doc.org/)"
         )
 
-if ( SPHINX_EXECUTABLE )
+if (SPHINX_EXECUTABLE)
     set(SPHINX_SOURCE ${CMAKE_CURRENT_SOURCE_DIR}/sphinx)
     set(SPHINX_BUILD ${CMAKE_CURRENT_BINARY_DIR}/sphinx)
 
     add_custom_target(sphinx
-            ${SPHINX_EXECUTABLE} -b html ${SPHINX_SOURCE} ${SPHINX_BUILD}/html
-            WORKING_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}
+            COMMAND ${CMAKE_COMMAND} -E env PYTHONPATH=${CMAKE_BINARY_DIR}:$ENV{PYTHONPATH} ${SPHINX_EXECUTABLE} -b html ${SPHINX_SOURCE} ${SPHINX_BUILD}/html
```

Comment:
Is the module written to `${CMAKE_BINARY_DIR}`? And wouldn't it be better to use `TEST_ENV` here as well?

Reply:
We might want to rename `TEST_ENV` to something more meaningful, like `ESP_PYTHON_ENV`.

Reply:
Down the road, not in this PR.

Reply:
> Is the module written to `${CMAKE_BINARY_DIR}`? And wouldn't it be better to use `TEST_ENV` here as well?

We need `_espressopp.so` which is located in the build dir and the python files which are located in the source dir under `src` but renamend to `espressopp`. The former is done by using the build dir the later by using a symlink.

Reply:
> We might want to rename `TEST_ENV` to something more meaningful, like `ESP_PYTHON_ENV`.

Yes, `TEST_ENV` does not tell what it is doing.